src/app/helptext/services/components/service-smb.ts

```diff
@@ -10,7 +10,8 @@ cifs_srv_netbiosname_tooltip: T('Automatically populated with the original hostn
 cifs_srv_netbiosname_validation : [ Validators.required, Validators.maxLength(15) ],
 
 cifs_srv_netbiosalias_placeholder: T('NetBIOS Alias'),
-cifs_srv_netbiosalias_tooltip: T('Enter an alias. Limited to 15 characters.'),
+cifs_srv_netbiosalias_tooltip: T('Enter any aliases, separated by spaces.\
+ Each alias cannot be longer than 15 characters.'),
```

Comment:
Invert the logic on this:
```suggestion
 Each alias can be up to 15 characters long.'),
```